src/ipa/rpi/common/ipa_base.cpp

```diff
@@ -484,7 +484,6 @@ void IpaBase::processStats(const ProcessParams &params)
 
 		/* reportMetadata() will pick this up and set the FocusFoM metadata */
 		rpiMetadata.set("focus.status", statistics->focusRegions);
-
```

Comment:
Usually we try to avoid making cosmetic changes to files (unless the cosmetic change was actually the purpose of the commit!), so we might consider putting the empty line back. I'm not terribly fussed, though...